docs/terminology.md

```diff
@@ -0,0 +1,62 @@
+# Overview
+This document contains all terminology used across compass documentation.
+
+## Management Plane 
+Abbr.: *MP*\
+\
+The multi-tenant system which allows to:
+- configure Applications
+- configure Runtimes
+- group Applications
+- assign the group of Applications to selected Runtimes
+
+### MP Connector
+Abbr.: *Connector*\
+\
+Component that establish trust among Applications, Management Plane and Runtimes.
+
+### MP Gateway
+Abbr.: *Gateway*\
+\
+Component that.. TBD
+
+### MP Director
+Abbr.: *Director*\
+\
+Component that.. TBD
+
+### MP Healtchecker
+Abbr.: *Healtchecker*\
+\
+Component that.. TBD
+
+### MP Tenant
+Abbr.: *Tenant*\
+\
+TBD
```

Comment:
do we want to merge this document wtih `TBD`?

Reply:
yes, I don't want to forget about those terms. And will update them soon or later.